daemon/graphdriver/windows/windows_test.go

```diff
@@ -0,0 +1,20 @@
+//+build windows
+
```

Comment:
Looks odd, but windows_windows_test.go would be consistent. eg pkg\system\syscall_windows_test.go & chtimes_windows_test.go